environment.yml

```diff
@@ -21,6 +21,7 @@ dependencies:
   - bioconda::genesplicer=1.0
   - bioconda::htslib=1.9
   - bioconda::manta=1.6.0
+  - bioconda::mapdamage2=2.2.0=pyr36_1
```

Comment:
What is this part at the end of the version number? `=pyr36_1`

Reply:
I copied it from the mapDamage [installation instructions](https://ginolhac.github.io/mapDamage/)

Reply:
I see, It's making the linting fail unfortunately.
And I'm noticing now that it's needing `python` > `3.5`, I'm afraid we'll get compatibilities issues with that.

Reply:
I've just had a look at anaconda and conda install -c bioconda mapdamage2 will should work

Reply:
OK, can you try and see if you can build the docker container with `mapdamage2` on your machine?